monkey/infection_monkey/exploit/log4shell_utils/ldap_server.py

```diff
@@ -94,7 +94,7 @@ def __init__(
         :param storage_dir: A directory where the LDAP server can safely store files it needs during
                             runtime.
         """
-        self._reactor_startup_completed = multiprocessing.Event()
+        self._reactor_startup_completed = multiprocessing.get_context("spawn").Event()
```

Comment:
I think we should leave a comment in the code as to why we're choosing spawn.

Nit: Create a context object and then we don't need to repeat "spawn"